settings.gradle

```diff
@@ -58,6 +59,8 @@ if (isEclipse) {
 
 include projects.toArray(new String[0])
 
+project(':build-tools').projectDir = new File(rootProject.projectDir, 'buildSrc')
```

Comment:
Can you not use the name `buildSrc` because that is reserved by gradle somehow? I think it would be nice for the names to line up or maybe have a comment why they don't.